yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
```

Comment:
I had to add the stuff in https://github.com/mapzen/vector-datasource/commit/6924d7a4236ecd347d87a02ed4ee2f0ea376eaed to get the tests to pass. The `extra_columns` thing is just so that the processor knows we're using the `height` column in the query, as it doesn't actually parse the query.

I didn't want to merge it in here directly, but the diff is a bit of a mess because I had to merge master to get a working install. Let me know what you think.